src/modules/Involvementapi.js

```diff
@@ -0,0 +1,24 @@
+const involvementAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const involvementAPIid = 'Q1L23qalAk2Wt8OSXMKp';
+
+function getAllLikes() {
+  return fetch(`${involvementAPI}apps/${involvementAPIid}/likes/`)
+    .then((response) => response.json())
+    .then((data) => data);
+}
+
+function postLikesImg(imageId) {
+  return fetch(`${involvementAPI}apps/${involvementAPIid}/likes/`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: imageId,
+    }),
+    headers: {
+      'content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => data);
+}
```

Comment:
This function is not part of your current task. Please try to do only the things specified in the task. That way you can focus on finding solutions for a single small problem at a time, and our code reviews will be much quicker.